### PR TITLE
Testing multiple clients and adding PHP 7. Closes #547

### LINKED ISSUE #547: Create TravisCI test matrix for v3 that uses Guzzle 5 and 6
We need to setup travis to somehow run our unit tests using Guzzle 5 and then Guzzle 6 for each supported PHP version.

### PR DESCRIPTION
**Pull request by mtdowling**

### CODE REVIEW

**Comment by jeremeamia** [comment]
Cool!

PHP7...

File: .travis.yml [hunk 1]
```diff
@@ -3,18 +3,24 @@ language: php
...
+  - COMPOSER_OPTS=""
+  - COMPOSER_OPTS="--prefer-lowest"
+
 matrix:
   allow_failures:
     - php: hhvm
+    - php: 7
   fast_finish: true
 
-sudo: false
```
Comment (GrahamCampbell) [inline .travis.yml]: ?

**Comment by mtdowling** [comment]
By counting the dots, I think it's segfaulting on test 90, which I think is Crc32ValidatingParserTest::testNothingWhenValidChecksum.

File: .travis.yml (hunk 1 above)
Comment (mtdowling) [inline .travis.yml]: I saw that in the flysystem .travis.yml file and copied it over. Does it not do anything?

Comment (GrahamCampbell) [inline .travis.yml]: It tells travis to use the new container architecture which is way faster. Old repos don't have it enabled by default on travis, so you must include it for now if you want the container architecture.

**Pull request merged**